osu.Game.Rulesets.Taiko/Scoring/TaikoScoreProcessor.cs

```diff
@@ -17,47 +16,13 @@ internal class TaikoScoreProcessor : ScoreProcessor<TaikoHitObject>
         /// </summary>
         private const double hp_hit_great = 0.03;
```

Comment:
We should probably rename this + the xml doc. How about `object_count_factor`?

Reply:
Along with that, would like to remove the 0.01 from hpMultiplier too, so I'd change this to be `1.0 / 3` to do so.